vs2019suneido/vs2019suneido.sln

```diff
@@ -44,8 +44,8 @@ Global
 		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|Win32.Build.0 = Debug|Win32
 		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|x64.ActiveCfg = Debug|x64
 		{FBE04237-9C7B-4973-9C60-505975998B39}.Debug|x64.Build.0 = Debug|x64
-		{FBE04237-9C7B-4973-9C60-505975998B39}.Profile|Win32.ActiveCfg = Release|Win32
-		{FBE04237-9C7B-4973-9C60-505975998B39}.Profile|Win32.Build.0 = Release|Win32
+		{FBE04237-9C7B-4973-9C60-505975998B39}.Profile|Win32.ActiveCfg = Profile|Win32

```

Comment:
I'm not sure if I'm reading this right, but it looks like Release|Win32 was replaced by Profile|Win32. Don't we still need Release|Win32?